packages/storybook/src/nimble/table/table.mdx

```diff
@@ -164,4 +164,12 @@ the current row have been traversed, `Tab`/`Shift-Tab` will move focus outside
 the table. When the table is focused again, navigation will resume from the
 previously focused element.
 
+### Selecting multiple rows
+
+When the table has a `selectionMode` of `multiple`, multiple rows can be
+selected using the selection checkboxes. Additionally, multiple rows can be
+selected by pressing the `CTRL` key while clicking each row, and a range of rows
+can be selected by pressing the `SHIFT` key while clicking on one row and then
+another.
```

Comment:
Can we describe the row interactions by breaking it down into four interactions:
- clicking a row: Selects just that row
- clicking a row followed by shift + click another row: Selects just that range of rows
- ctrl + click on a row: Adds additional single rows to selection
- ctrl + click on a row followed by ctrl + shift + click on a different row: Adds a range of rows to the selection